modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/search/helper/DDMSearchUtil.java

```diff
@@ -63,10 +59,11 @@ public SearchContext buildStructureLayoutSearchContext(
 		return searchContext;
 	}
 
-	public SearchContext buildStructureSearchContext(
+	public static SearchContext buildStructureSearchContext(
 		long companyId, long[] groupIds, long userId, long classNameId,
 		Long classPK, String name, String description, String storageType,
 		Integer type, int status, int start, int end,
+		DDMPermissionSupport ddmPermissionSupport,
```

Comment:
Hey @MrJohn1911, just pass the result of the `ddmPermissionSupport.getStructureModelResourceName(classNameId)` if possible. Thx